govdelivery-subscribe/error/index.html

```diff
@@ -5,17 +5,28 @@
 {%- endblock %}
 
 {% block desc -%}
-    Your subscription was not successfuly received.
+    Your subscription was not successfully received.
 {%- endblock %}
 
 {% block content_main %}
+    <h1><span class="cf-icon cf-icon-error-round"></span> Not just yet!</h1>
+    <p class="short-desc">
+        We’re still building the email signup function for <br>
+        this beta release. It will be available soon. For now:
+    </p>
+    <ol class="short-desc">
+        <li>Keep exploring <a href="http://beta.consumerfinance.gov">beta.consumerfinance.gov</a>.</li>
+        <li>Check back often to follow our progress.</li>
+        <li>Sign up for general CFPB updates at <a href="http://www.consumerfinance.gov">www.consumerfinance.gov</a>.</li>
+    </ol>
+    <!-- Commented out until email signup is ready to use.
```

Comment:
You might add `TODO:` to the code comment, like `<!-- TODO: Commented...` so we can search for `TODO` comments and see what still needs to be fixed in the code if not otherwise covered by issues or tasks.